### DISCUSSION TITLE
teoria relazionale - 2

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#69] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/0bed9bb8-a507-44d1-8616-e6309c7be6b4)

**Comment by daromacarlo** [comment]
![Immagine 2024-02-04 183056](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/cef70394-4a24-4ff0-aafd-adea9f422d64)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/79638284/0239ef45-f261-4515-8b3e-b32c027c4190)
